Applications/Custom/LayerPlugin/layer_plugin_test.cpp

```diff
@@ -65,11 +65,10 @@ TEST(AppContext, DlRegisterDirectory_n) {
 TEST(AppContext, DefaultEnvironmentPath_p) {
   /// as NNTRAINER_PATH is fed to the test, this should success without an
   /// error
-  auto l = ml::train::createLayer("pow");
+  std::shared_ptr<ml::train::Layer> l = ml::train::createLayer("pow");
   EXPECT_EQ(l->getType(), "pow");
 
-  std::unique_ptr<nntrainer::Layer> layer(
-    static_cast<nntrainer::Layer *>(l.release()));
+  auto layer = nntrainer::getLayerDevel(l);
```

Comment:
getLayerDevel will be removed from layer_internal.h (to be renamed to layer_devel.h soon), and will only be available for internal unittests.